src/main/java/com/google/sps/servlets/CasesDataServlet.java

```diff
@@ -64,7 +64,9 @@ public void init() {
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setCharacterEncoding(ENCODING);
     response.setContentType(CTYPE);
-    response.getWriter().println(reportsJson);
+    if (reportsJson != null) {
+      response.getWriter().println(reportsJson);
```

Comment:
How to test this? Can you use this example: https://stackoverflow.com/questions/5434419/how-to-test-my-servlet-using-junit? Feel free to find more examples online.